src/providers/MeetingProvider/docs/MeetingManager.stories.mdx

```diff
@@ -84,6 +84,7 @@ For example - you may want to fetch the attendee from the database, or get the n
 
 interface AttendeeResponse {
   name?: string;
+  [attribute: string]: any;
```

Comment:
Should this be an optional field?

Reply:
It is an optional field right now. User could only put `name` to `AttendeeResponse`. 
I do think it would be better to somehow mark it as optional. I tried to add a question mark like this `[attribute: string]?: any;`, but it's invalid to do that. Do you know any way to achieve this?

Reply:
Actually, this should be fine. I don't think you need that '?' - it should be optional already